packages/cli/src/utils/analytics.js

```diff
@@ -20,7 +20,7 @@ const shouldSkipAnalytics = (mode) =>
   process.env.DISABLE_ZAPIER_ANALYTICS ||
   mode === ANALYTICS_MODES.disabled;
 
-const recordAnalytics = async (command, isValidCommand, args, flags) => {
+const recordAnalytics = async (command, isValidCommand, argNames, flags) => {
```

Comment:
`recordAnalytics` is called at [ZapierBaseCommand.js](https://github.com/zapier/zapier-platform/pull/890/files#diff-3b23b2f6b3353d797bd2412d62eea92f69d33a48ef839325edea158cb4c432b9R349).`Object.keys(this.args)` returns an array of argument names rather than the arguments per se.